### PR TITLE
bugfix/ local storage allows user to leave webpage and come back

### PR DESCRIPTION
**Pull request by b-stein**
#### What's this PR do?
moves the local storage to set an item within the componentDidUpdate life cycle method

#### Where should the reviewer start?
files changed tab

#### How should this be manually tested?
UI

### CODE REVIEW

File: src/App/App.js [hunk 1]
```diff
@@ -10,40 +10,48 @@ import { getAllMovies, getUserRatedMovies } from '../apiCalls/apiCalls';
 class App extends Component {
 	constructor() {
 		super();
-		
-		let savedState = JSON.parse(localStorage.getItem('localStorageState'));
+		let loggedUserId = JSON.parse(localStorage.getItem('loggedInUserId'));
+		let loggedUserName = JSON.parse(localStorage.getItem('loggedInUserName'));
+		let loggedRatings = JSON.parse(localStorage.getItem('loggedRatings'));
```
Comment (b-stein) [inline src/App/App.js:15]: maybe not the best place to leave this logic

**Comment by b-stein** [comment]
@attom2

File: src/App/App.js [hunk 2]
```diff
@@ -10,40 +10,48 @@ import { getAllMovies, getUserRatedMovies } from '../apiCalls/apiCalls';
...
 		} else if (!this.state.userID) {
 			this.props.history.push('/');
 		}
 	}
+	
+	componentDidUpdate = () => {
+		let loggedUserId = JSON.stringify(this.state.userID);
+		let loggedUserName = JSON.stringify(this.state.userName);
+		let loggedRatings = JSON.stringify(this.state.userRatings);
+
```
Comment (attom2) [inline src/App/App.js:50]: Good use of Lifecycle updating phase!

File: src/App/App.js (hunk 1 above)
Comment (attom2) [reply src/App/App.js:15]: We could try moving it to componentDidMount or componentDidUpdate

**Pull request merged**